### PR TITLE
Better 0.2.0

### PR DESCRIPTION
**Pull request by ManiacMoustache**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Steps to Test
1. Test hopper/loader logic and shooting

## Changelog
Removed hopper backdown and slowed down speeds.
Raised drivetrain multiplier to move faster

## Why?
To shoot and drive faster

### CODE REVIEW

File: components/Actuators/HighLevel/loaderLogic.py [hunk 1]
```diff
@@ -78,12 +78,12 @@ def runHopperManually(self):
     @state
     def checkForBall(self):
         """Checks for ball to enter the loader, runs the loader if entry sensor is broken."""
-        ballCount = self.ballCounter.getBallCount()
+        # ballCount = self.ballCounter.getBallCount()
```
Comment (ChrisHirsch) [inline components/Actuators/HighLevel/loaderLogic.py:81]: Remove comments

**Review by ChrisHirsch — CHANGES_REQUESTED** [review]
Tunables all look good
Remove the commented code

**Review by hell0w0rld2 — APPROVED** [review]

File: components/Actuators/HighLevel/shooterLogic.py [hunk 2]
```diff
@@ -97,7 +98,8 @@ def isShooterUpToSpeed(self):
                 and abs(self.shooterMotor2Encoder.getVelocity() - shootSpeed2) <= abs(self.shootTolerance))
         rumble  = 0
         if atSpeed and not self.isAutonomous:
-            rumble = .3
+            rumble = 1
+        logging.error("1: %s vs %s, 2: %s vs %s", shootSpeed1, self.shooterMotor1Encoder.getVelocity(), shootSpeed2, self.shooterMotor2Encoder.getVelocity())
```
Comment (ChrisHirsch) [inline components/Actuators/HighLevel/shooterLogic.py:102]: Won't this log (and do getVelocity()) on every frame? Could this be what's slowing us down?

**Review by ChrisHirsch — CHANGES_REQUESTED** [review]
Unless needed, lets remove the commented code and *especially* consider removing thowing an error log every frame. This logs back at the driver station right?

**Comment by ManiacMoustache** [comment]
good to go

**Review by ChrisHirsch — APPROVED** [review]
+💯 Go kill it!

**Pull request merged**